### PR TITLE
style(runloop/wasm): style clean with early return

### PR DESCRIPTION
**Pull request by chronolaw**
### Summary

style clean with early return

### Checklist

- [ ] The Pull Request has tests
- [x] A changelog file has been created under `changelog/unreleased/kong` or `skip-changelog` label added on PR if changelog is unnecessary. [README.md](https://github.com/Kong/gateway-changelog/blob/main/README.md)
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Issue reference

Fix #_[issue number]_

### CODE REVIEW

File: kong/runloop/wasm.lua [hunk 1]
```diff
@@ -231,24 +231,21 @@ end
...
 
-  elseif typ == TYPE_COMBINED and service_id and route_id then
+  if typ == TYPE_COMBINED and service_id and route_id then
     local routes = state.combined[service_id]
-    ref = routes and routes[route_id]
-
-  else
-    -- unreachable
-    error("unknown filter chain type: " .. tostring(typ), 2)
+    return routes and routes[route_id]
```
Comment (bungle) [inline kong/runloop/wasm.lua:244]: I would probably prefer `if..elseif..else` but you can remove the local ref at the top and return ref at the bottom.

File: kong/runloop/wasm.lua [hunk 2]
```diff
@@ -268,14 +265,18 @@ local function store_chain_ref(state, ref)
...
+  if typ == TYPE_ROUTE then
     assert(type(route_id) == "string",
            ref.label .. " chain has no route ID")
 
     state.by_route[route_id] = ref
+    return
+  end
 
-  elseif typ == TYPE_COMBINED then
+  if typ == TYPE_COMBINED then
```
Comment (bungle) [inline kong/runloop/wasm.lua:279]: Here I think the original was better.

**Review by bungle — CHANGES_REQUESTED** [review]
Just how I see this.

**Comment by chronolaw** [comment]
Since it is only a style clean, if we think the original is better then we need not to change them.

**Pull request closed without merging**